src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -655,6 +678,22 @@ typedef struct {
     // Indicates which candidates the ICE Agent is allowed to use.
     ICE_TRANSPORT_POLICY iceTransportPolicy;
 
+    // Set of certificates that the RtcPeerConnection uses to authenticate.
+    // Although any given DTLS connection will use only one certificate, this
+    // attribute allows the caller to provide multiple certificates that support
+    // different algorithms.
+    //
+    // If this value is absent, then a default set of certificates is generated
+    // for each RtcPeerConnection.
+    //
+    // !!!!!!!!!! IMPORTANT !!!!!!!!!!
+    // It is recommended to rotate the certificates often - preferrably for every peer connection
+    // to avoid a compromised client weakening the security of the new connections
+    PRtcCertificate certificates; // Max MAX_RTCCONFIGURATION_CERTIFICATES certificates
```

Comment:
Can we do `PRtcCertificate certificates[MAX_RTCCONFIGURATION_CERTIFICATES + 1]` instead? Having to set both members is error prone.

Then later we can do
```
for (UINT32 i = 0; i < MAX_RTCCONFIGURATION_CERTIFICATES; i++) {
  if (certificates[i] ==NULL) {
    break;
  }
}
```

Reply:
Agree, changed to an array